frontend/src/components/SingleChat.js

```diff
@@ -100,6 +104,14 @@ const SingleChat = ({ fetchAgain, setFetchAgain }) => {
       }
     }
   };
+const handleEmojiClick = (emojiObject) => {
+  setNewMessage(newMessage + emojiObject.emoji); // Correct usage to append emoji
+};
+
+const toggleEmojiPicker = () => {
+  setShowEmojiPicker(!showEmojiPicker); // Toggle the visibility of emoji picker
```

Comment:
can you do like this `setShowEmojiPicker(emoji => !emoji)`

Reply:
okay let me correct it